spec/version.dd

```diff
@@ -294,6 +295,7 @@ $(H3 $(LEGACY_LNAME2 PredefinedVersions, predefined-versions, Predefined Version
 	$(TROW $(ARGS $(D HPPA)) , $(ARGS The HP PA-RISC architecture, 32-bit))
 	$(TROW $(ARGS $(D HPPA64)) , $(ARGS The HP PA-RISC architecture, 64-bit))
```

Comment:
Btw while we are at this list looks pseudo-sorted:
- SystemZ
- HPPA
- SH
...